testing/integration/k8s/testdata/opentelemetry-kube-stack/README.md

```diff
@@ -139,6 +139,11 @@ Consult also the [Helm Documentation on CRDs](https://helm.sh/docs/chart_best_pr
 
 _See [helm upgrade](https://helm.sh/docs/helm/helm_upgrade/) for command documentation._
 
+### Upgrade from 0.6.x to 0.7.x
+
+Version 0.7.0 has unified the previous collectors (daemonset and deployment) in a single one. If you are using custom configurations for `cluster` collector, you will need to merge your `cluster` collector configuration with `daemon` collector and remove `collectors.cluster` section from your values file.
```

Comment:
Does this mean that now with this change we will be collecting cluster level stats through the leader of the Daemonset? If that's the case, I'm not sure if this should change without sufficient benchamarking.

See https://www.elastic.co/docs/reference/fleet/scaling-on-kubernetes#configuration-practices for more information on why the Leader Election might not be the best option for some large use-cases.